src/frontend/src/components/Eligibility/index.tsx

```diff
@@ -51,6 +51,12 @@ export default class Eligibility extends React.Component<Props> {
     };
 
     const eligibility = () => {
+
+      // Time ineligibility without a date (meaning "never eligible") beats all other rules for eligibility
+      if (time_eligibility && time_eligibility.status === 'Ineligible' && time_eligibility.date_will_be_eligible == null) {
+        return ineligible;
+      }
+
```

Comment:
How does this affect line 46 of this file (hit the expand arrow above on Github to see this)?

Reply:
Excellent question. That line will no longer ever run. I considered taking it out now, but my thought was to make that part of getting rid of most of the case-checking logic from this section in the frontend, as described in the PR description, in a future PR.

Reply:
I believe it was never run before too, unless there was a bug in the backend logic. I'm trying to think how this can be integrated into the switch statement below.